classes/village/VillageBuildingConfig.php

```diff
@@ -35,263 +59,275 @@ class VillageBuildingConfig {
+        self::BUILDING_HOSPITAL => "Dedicated to the swift recovery of its warriors, the hospital stands as a beacon of resilience and care within the village.",
+        self::BUILDING_ANBU_HQ => "Command center for the village's military might, the ANBU HQ coordinates training and deployment of elite operatives to safeguard the village from unseen threats.",
+        self::BUILDING_MARKETPLACE => "A bustling heart of commerce, the marketplace is a vibrant tapestry of traders and goods ensuring that the needs of the village are met.",
+        self::BUILDING_SHRINE => "A sacred space of solace and spirituality, the shrine connects the physical to the divine, offering a place for reflection, reverence, and ritual.",
+        self::BUILDING_RAMEN_STAND => "More than just a place to eat, the ramen stand is a culinary refuge where delicious flavors and warm conversations nourish both body and soul.",
     ];
 
-    /* building poetic phrase indexed by building ID */
+    /* building poetic phrase indexed by building key */
     const BUILDING_PHRASE = [
-        self::BUILDING_VILLAGE_HQ => "The heart of the village.",
-        self::BUILDING_WORKSHOP => "The hands of the village.",
-        self::BUILDING_ACADEMY => "The mind of the village.",
-        self::BUILDING_HOSPITAL => "The soul of the village.",
-        self::BUILDING_ANBU_HQ => "The shadow of the village.",
-        self::BUILDING_MARKET => "The lifeblood of the village.",
-        self::BUILDING_SHRINE => "The spirit of the village.",
-        self::BUILDING_RAMEN_STAND => "The belly of the village.",
+        self::BUILDING_VILLAGE_HQ => "The heart from which all paths are drawn.",
+        self::BUILDING_WORKSHOP => "Where ambition meets foundation.",
+        self::BUILDING_ACADEMY => "Today's students, tomorrow's masters.",
+        self::BUILDING_HOSPITAL => "Where warriors mend and strength renews.",
+        self::BUILDING_ANBU_HQ => "Watchful eyes and honed blades.",
+        self::BUILDING_MARKETPLACE => "Where coin flows, fortune follows.",
+        self::BUILDING_SHRINE => "Hushed prayers and solemn offerings.",
+        self::BUILDING_RAMEN_STAND => "A bowl of warmth, a friendly smile.",
     ];
 
-    /* construction cost indexed by building ID, then tier, then resource type */
+    /* construction cost indexed by building key, then tier, then resource type */
     const BUILDING_CONSTRUCTION_COST = [
         self::BUILDING_VILLAGE_HQ => [
             1 => [
-                WarManager::RESOURCE_MATERIALS => 10800,
-                WarManager::RESOURCE_FOOD => 4320,
-                WarManager:: RESOURCE_WEALTH => 6480,
+                WarManager::RESOURCE_MATERIALS => self::BASE_T1_CONSTRUCTION_COST_TOTAL * 0.5 * 1.5,
+                WarManager::RESOURCE_FOOD => self::BASE_T1_CONSTRUCTION_COST_TOTAL * 0.2 * 1.5,
+                WarManager::RESOURCE_WEALTH => self::BASE_T1_CONSTRUCTION_COST_TOTAL * 0.3 * 1.5,
```

Comment:
What is the `* 1.5` modifier representing here?

Reply:
Making some buildings have a higher or lower total construction time/cost. VillageHQ for example, 50% more expensive while Ramen Stand is 50% less expensive.